website/playground/src/utils/UrlSharer.ts

```diff
@@ -65,15 +80,19 @@ export class UrlSaver {
     }
   }
 
-  updateUrl({ text, configText, language }: {
+  updateUrl({ text, configText, plugin, language }: {
     text: string;
     configText?: string;
+    plugin?: string;
     language?: "typescript" | "json" | "markdown" | "toml" | "dockerfile";
   }) {
     let url = `#code/${compressToEncodedURIComponent(text)}`;
     if (configText != null) {
       url += `/config/${compressToEncodedURIComponent(configText)}`;
     }
+    if (plugin != null) {
```

Comment:
Maybe we should avoid doing this if the plugin is a default plugin? Also, maybe we shouldn't append a language when the plugin is set?

Reply:
I've updated but added the logic outside of the URL saver.